### PR TITLE
GIT24440: In step 5, for the example I changed the minimum to 1. Also…

### LINKED ISSUE #24440: [enterprise-3.11] Issue in file admin_guide/cluster-autoscaler.adoc
### Which section(s) is the issue in?

Creating the launch configuration and Auto Scaling group

### What needs fixing?

We write that the asg may use min 0 nodes - but the autoscaler provided by Red Hat needs at least 1 node as template to function. So step5 should NOT be:

```
aws autoscaling create-auto-scaling-group \
      --auto-scaling-group-name mycluster-ASG \ 
      --launch-configuration-name mycluster-LC \ 
      --min-size 0 \ 
      --max-size 6 \ 
      --vpc-zone-identifier subnet-12345678 \ 
      --tags ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=Name,Value=mycluster-ASG-node,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=kubernetes.io/cluster/mycluster,Value=true,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute,Value=true,PropagateAtLaunch=true
```

but

```
aws autoscaling create-auto-scaling-group \
      --auto-scaling-group-name mycluster-ASG \ 
      --launch-configuration-name mycluster-LC \ 
      --min-size 1 \ 
      --max-size 6 \ 
      --vpc-zone-identifier subnet-12345678 \ 
      --tags ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=Name,Value=mycluster-ASG-node,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=kubernetes.io/cluster/mycluster,Value=true,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute,Value=true,PropagateAtLaunch=true 
```
instead. The --min-size needs to be 1 and not 0.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
Fixes #24440

Changed step 5 example to "min=1" and added clarification to the description that at least one node is required for the template.

### CODE REVIEW

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc [hunk 1]
```diff
@@ -90,14 +90,14 @@ NOTE: If your template is fewer than 16 KB before you encode it, you can provide
...
-      --min-size 0 \ <3>
+      --min-size 1 \ <3>
       --max-size 6 \ <4>
       --vpc-zone-identifier subnet-12345678 \ <5>
       --tags ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=Name,Value=mycluster-ASG-node,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=kubernetes.io/cluster/mycluster,Value=true,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute,Value=true,PropagateAtLaunch=true <6>
 ----
 <1> Specify the name of the Auto Scaling group, which you use when you deploy the auto-scaler deployment
 <2> Specify the name of the Launch Configuration that you created.
-<3> Specify the minimum number of nodes that the auto-scaler maintains.
+<3> Specify the minimum number of nodes that the auto-scaler maintains. At least one node is required for the template to function.
```
Comment (pneedle-rh) [inline admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc:100]: I would recommend shortening the second sentence to:

"At least one node is required."

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc [hunk 2]
```diff
@@ -90,14 +90,14 @@ NOTE: If your template is fewer than 16 KB before you encode it, you can provide
...
 <4> Specify the maximum number of nodes the scale group can expand to.
 <5> Specify the VPC subnet-id, which is the same subnet that the cluster uses.
-<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
```
Comment (pneedle-rh) [inline admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc:103]: Can you please check why this line is being listed as changed? I can't see any differences here. Thank you!

**Review by pneedle-rh — COMMENTED** [review]
@jonquilwilliams this looks good to me, aside from a couple of small suggestions.

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc:103]: Hi, Paul. I didn't deliberately make a change here. Perhaps I accidentally hit a key and didn't undo it properly. This line should be fine as is.

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc]: I have made these changes and pushed the changes to git. Thanks for the suggestion.

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc (hunk 2 above)
Comment (pneedle-rh) [reply admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc:103]: Ok thank you. Just for reference, looking at `git show` for your commit, I think the change relates to a new line having been added to the end of the file:

```
-<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
\ No newline at end of file
+<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
```

File: admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc (hunk 1 above)
Comment (pneedle-rh) [reply admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc]: Thank you!

**Comment by pneedle-rh** [comment]
@jonquilwilliams this looks good to me. Please request QE review for this PR. Thanks!

**Comment by jonquilwilliams** [comment]
@sunzhaohua2 Please review this PR so that it can be merged. Thanks!

**Pull request merged**